website/content/docs/connect/cluster-peering/k8s.mdx

```diff
@@ -177,9 +177,9 @@ Peers identify each other using the `metadata.name` values you establish when cr
 
 The examples described in this section demonstrate how to export a service named `backend`. You should change instances of `backend` in the example code to the name of the service you want to export.
 
-1. For the service in `cluster-02` that you want to export, add the following [annotations](/docs/k8s/annotations-and-labels) to your service's pods. 
+1. For the service in `cluster-02` that you want to deploy onto the mesh and evautally export, ensure that you add the `"consul.hashicorp.com/connect-inject": "true"` annotation to your service's pods prior to deploying. Below is an example workload with the annotation highlighted that allows the workload to join the mesh. 
```

Comment:
```suggestion
1. For the service in `cluster-02` that you want to export, add the `"consul.hashicorp.com/connect-inject": "true"` annotation to your service's pods prior to deploying. The annotation allows the workload to join the mesh. It is highlighted in the following example: 
```
I think we might be packing too much workflow detail into this step. Could we add this to the introduction paragraph?